docs/blog/2021-12-09-log4j-zero-day.md

```diff
@@ -59,6 +59,10 @@ Many Open Source projects
 like the Minecraft server, [Paper](https://github.com/PaperMC/Paper/commit/b475c6a683fa34156b964f751985f36a784ca0e0),
 have already begun patching their usage of `log4j2`.
 
+This [proof of concept](https://twitter.com/chvancooten/status/1469340927923826691) of changing an iPhone's name demonstrates
+that physical devices are also affected by this vulnerability. To be clear, this proof of concept only shows the vulnerability
+exists on iPhones, but there is currently no known remote method of triggering it.
+
```

Comment:
erm, no, this demonstrates the vulnerability exists somewhere on a server operated by Apple